node_modules/gh-users/lib/rest.js

```diff
@@ -107,6 +107,7 @@ GrassHopper.appRouter.on('get', '/api/me', function(req, res) {
     if (req.ctx.user) {
         me = _.extend(me, {'anon': false}, req.ctx.user.toJSON());
     }
+    me = _.extend(me, {'termsAndConditions': UsersAPI.getTermsAndConditionsStatus(req.ctx)});
```

Comment:
I'm suggesting to change it to:

`_.extend(me, {'termsAndConditions': UsersAPI.getTermsAndConditionsStatus(req.ctx)});`

Rather than do the assign (`=`) to the `me` variable.